enterprise/frontend/src/embedding-sdk/components/public/dashboard/SdkDashboard.tsx

```diff
@@ -227,6 +243,15 @@ const SdkDashboardInner = ({
         dispatch(setEditingDashboard(dashboard));
         dispatch(toggleSidebar(SIDEBAR_NAME.addQuestion));
       }}
+      // We only want the dashboard view to be rerendered, so the new dashboard questions are loaded after being created.
+      setRefetchDashboard={(refetchDashboard) => {
+        refetchDashboardRef.current = refetchDashboard;
+      }}
+      autoScrollToDashcardId={
+        dashboard?.dashcards.find(
```

Comment:
Doing the exact same thing as the core app. Maybe there's a better way to do this 🤷 

https://github.com/metabase/metabase/blob/051b61bd256c2337880f1354e69cb6a4dca2dbbc/frontend/src/metabase/query_builder/components/MoveQuestionModal/MoveQuestionModal.tsx#L99-L111